bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/spinner/SpinnerAjaxButton.java

```diff
@@ -90,21 +91,32 @@ public LaddaAjaxButton setEffect(LaddaBehavior.Effect effect) {
      * @param color The color for the spinner
      * @return {@code this}, for chaining
      */
-    public LaddaAjaxButton setSpinnerColor(LaddaBehavior.Color color) {
-        this.laddaBehavior.withSpinnerColor(color);
+    public SpinnerAjaxButton setSpinnerColor(SpinnerBehavior.Color color) {
+        this.spinnerBehavior.withSpinnerColor(color);
+        return this;
+    }
+
+    /**
+     * Sets if label should be hidden
+     *
+     * @param noLabel <code>true</code> if label should be hidden
+     * @return {@code this}, for chaining
+     */
+    public SpinnerAjaxButton setEffect(boolean noLabel) {
```

Comment:
This is a new method, right ?
Why its name is `setEffect` ? Maybe `setNoLabel` or `showLabel() + hideLabel()` ?!

Reply:
my bad, missed it :((